packages/restapi/src/lib/spaceV2/inviteToJoin.ts

```diff
@@ -0,0 +1,191 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import * as Peer from 'simple-peer';
+import { produce } from "immer";
+
+import { SpaceV2 } from "./SpaceV2";
+import { SpaceInviteInputOptions } from "./types";
+
+import getIncomingIndexFromAddress from "../video/helpers/getIncomingIndexFromAddress";
+import { VideoCallStatus } from "../types";
+import { getIceServerConfig } from "../video/helpers/getIceServerConfig";
+import sendSpaceNotification from './helpers/sendSpaceNotification';
+import { VIDEO_CALL_TYPE } from '../payloads/constants';
+import isJSON from '../video/helpers/isJSON';
+
+export async function inviteToJoin(
+    this: SpaceV2, 
+    options: SpaceInviteInputOptions
```

Comment:
imo having the interfaces for independent functions in the function declaration file itself is helpful while writing and maintaining code as we dont have to switch files, as well ~15 lines of code doesnt pollute the file as well. so lets keep the Interface / Type declaration of the function in the file itself.